tests/native_tests/language.hy

```diff
@@ -1221,9 +1219,11 @@
   "NATIVE: test if keywords are recognised"
 
   (assert (= :foo :foo))
+  (assert (= :foo ':foo))
```

Comment:
You might also test `(is (type :foo) (type ':foo))`.

Reply:
Done